systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -87,19 +88,22 @@
     protected static final String TLS_SIDECAR_KAFKA_IMAGE = "STRIMZI_DEFAULT_TLS_SIDECAR_KAFKA_IMAGE";
     protected static final String TLS_SIDECAR_EO_IMAGE = "STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE";
     protected static final String TEST_TOPIC_NAME = "test-topic";
-    protected static final String CONSUMER_GROUP_NAME = "my-consumer-group";
 
     protected String testClass;
     protected String testName;
 
     protected Random rng = new Random();
 
     public static final int MESSAGE_COUNT = 100;
+
     public static final String TOPIC_NAME = KafkaTopicUtils.generateRandomNameOfTopic();
     public static final String EXAMPLE_TOPIC_NAME = "my-topic";
-
+    public static final String AVAILABILITY_TOPIC_SOURCE_NAME = "availability-topic-source-" + new Random().nextInt(Integer.MAX_VALUE);
+    public static final String AVAILABILITY_TOPIC_TARGET_NAME = "availability-topic-target-" + new Random().nextInt(Integer.MAX_VALUE);
```

Comment:
ditto as above, I see you have a `rng` instance, please use it to generated random numbers and not new instances of `Random`.